docker/edge/http/script.sh

```diff
@@ -153,16 +174,29 @@ if [ $? -eq 0 ]; then
    fi
 done
 
```

Comment:
[minor]This section code generate token for IOT Device, its not part of edge. Write below comment above it.
#Below authorization token should be generated by the user only
#Below code is written for Demo purpose
#Let assume the username is "admin@test.com" and passowrd is "1234"

Reply:
These comments are successfully included in the script.